run.py

```
@@ -0,0 +1,813 @@
+            os.system('start relive_tv.py ' + text)
+            time.sleep(35)
+            countshit += 1
+    if countshit == 5:
+            logging.info("failed shutdown script")
+            os.system('TASKKILL /f /im ' + ffmpeg)
+            os.system('start relive_tv.py ' + text)
+            exit()
+
+def checktitlelol(arg1, arg2, reload, url_omg):
+      if config.Twitch == "True":
+        if reload == "Null":
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)